biosimulators_utils/config.py

```diff
@@ -147,8 +160,8 @@ def __init__(self,
         self.COLLECT_COMBINE_ARCHIVE_RESULTS = COLLECT_COMBINE_ARCHIVE_RESULTS
         self.COLLECT_SED_DOCUMENT_RESULTS = COLLECT_SED_DOCUMENT_RESULTS
         self.SAVE_PLOT_DATA = SAVE_PLOT_DATA
-        self.REPORT_FORMATS = REPORT_FORMATS
-        self.VIZ_FORMATS = VIZ_FORMATS
+        self.REPORT_FORMATS = REPORT_FORMATS or [ReportFormat.csv]
```

Comment:
Hah, I saw this and thought it was an actual csv file rather than a csv attribute of `ReportFormat`.

Reply:
You know, I did at first as well! As I mentioned, I moved the logic for this inside the actual constructor as per `SonarCloud`.